cluster-autoscaler/simulator/delta_cluster_snapshot.go

```diff
@@ -51,24 +49,24 @@ type deltaSnapshotPodLister DeltaClusterSnapshot
 type internalDeltaSnapshotData struct {
 	baseData *internalDeltaSnapshotData
 
-	addedNodeInfoMap    map[string]*schedulernodeinfo.NodeInfo
-	modifiedNodeInfoMap map[string]*schedulernodeinfo.NodeInfo
+	addedNodeInfoMap    map[string]*schedulerframework.NodeInfo
+	modifiedNodeInfoMap map[string]*schedulerframework.NodeInfo
 	deletedNodeInfos    map[string]bool
 
-	nodeInfoList         []*schedulernodeinfo.NodeInfo
+	nodeInfoList         []*schedulerframework.NodeInfo
 	podList              []*apiv1.Pod
-	havePodsWithAffinity []*schedulernodeinfo.NodeInfo
+	havePodsWithAffinity []*schedulerframework.NodeInfo
```

Comment:
Is podList still used for anything after removing the pod lister functionality?

Reply:
No, good catch - fixed.